apps/web/src/design-system/ThemeProvider.tsx

```diff
@@ -40,37 +41,39 @@ export function ThemeProvider({ children }: { children: JSX.Element; dark?: Bool
       <MantineProvider
         withGlobalStyles
         withNormalizeCSS
-        styles={{
-          Notification: (theme) => ({
-            root: {
-              backgroundColor: theme.colorScheme === 'dark' ? colors.B15 : colors.white,
-              border: 'none',
-              boxShadow: theme.colorScheme === 'dark' ? shadows.dark : shadows.light,
-              height: '65px',
-              width: '460px',
-              borderRadius: '7px',
-              padding: '20px',
-              justify: 'space-between',
-            },
-            description: {
-              fontSize: '16px',
-              fontWeight: '400',
-              color: theme.colorScheme === 'dark' ? colors.white : colors.B40,
-            },
-            closeButton: {
-              color: theme.colorScheme === 'dark' ? colors.B40 : colors.B80,
-            },
-            icon: {
-              width: '22px',
-              height: '22px',
-              marginRight: '10px',
-            },
-          }),
-        }}
         theme={{
           // Override any other properties from default theme
           colorScheme,
           ...mantineConfig,
+          components: {
+            Notification: {
+              styles: (theme) => ({
+                root: {
+                  backgroundColor: theme.colorScheme === 'dark' ? colors.B15 : colors.white,
+                  border: 'none',
+                  boxShadow: theme.colorScheme === 'dark' ? shadows.dark : shadows.light,
+                  height: '65px',
+                  width: '460px',
+                  borderRadius: '7px',
+                  padding: '20px',
+                  justify: 'space-between',
+                },
+                description: {
+                  fontSize: '16px',
+                  fontWeight: 400,
+                  color: theme.colorScheme === 'dark' ? colors.white : colors.B40,
+                },
+                closeButton: {
+                  color: theme.colorScheme === 'dark' ? colors.B40 : colors.B80,
+                },
+                icon: {
+                  width: '22px',
+                  height: '22px',
+                  marginRight: '10px',
+                },
+              }),
+            },
```

Comment:
As you did in other components you could use `useStyles` from Mantine to extract the CSS. I think the lines I highlighted could be done.